src/lib/common/tag.h

```diff
@@ -130,7 +128,6 @@ extern std::string valueTag
   const std::string&  value,
   Format              format,
   bool                showComma           = false,
-  bool                isAssociation       = false,
   bool                valueIsNumberOrBool = false
 );
 
```

Comment:
ok, I didn't see the entire picture in the diff here in guthub.
Sorry for not taking the time to open the file before I commented.
It is clear that there can be no compiler-confusion.

NTC
NTC